datumaro/plugins/open_images_format.py

```diff
@@ -337,6 +372,89 @@ def _load_bboxes(self, items_by_id):
                         attributes=attributes,
                     ))
 
+    def _load_masks(self, items_by_id):
+        label_categories = self._categories[AnnotationType.label]
+
+        for mask_path in self._glob_annotations(
+            '*' + OpenImagesPath.MASK_DESCRIPTION_FILE_SUFFIX
+        ):
+            with self._open_csv_annotation(mask_path) as mask_reader:
+                for mask_description in mask_reader:
+                    mask_path = mask_description['MaskPath']
+                    if _RE_INVALID_PATH_COMPONENT.fullmatch(mask_path):
```

Comment:
How about `osp.abspath().startswith(dataset_root + osp.sep)`?

Reply:
That's too forgiving. Something like `../../masks/validation/foobar.png` can pass a check like that, but it should not be accepted as a valid path.